src/home/NotificationCenter.tsx

```diff
@@ -223,6 +258,8 @@ export default function Notifications({ navigation }: NotificationsProps) {
   )
 }
 
+export const listGapHeight = Spacing.Thick24
```

Comment:
Looks like this export is only needed for testing. 

Could you change it to:
```suggestion
// Exported for testing
export const _listGapHeight = Spacing.Thick24
```

Reply:
I'm inclined to revert this as well. Instead, I'll provide heights for simulation directly in the test.